### PR TITLE
Add release builds that produce symbols for compliance scans

### PR DESCRIPTION
**Pull request by TravisEz13**
Add two new named release builds that produce zip files with symbols.
update the infrastructure to support these builds.
Add a readme to describe how to use the release builds.

### CODE REVIEW

File: tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1 [hunk 1]
```diff
@@ -55,16 +56,30 @@ try{
...
-    Start-PSPackage @pspackageParams @releaseTagParam
+    if(!$Symbols.IsPresent)
+    {
+        Write-Verbose "Starting powershell packaging(msi)..." -verbose
+        Start-PSPackage @pspackageParams @releaseTagParam
+    }
+    else 
+    {
+        $pspackageParams += @{'IncludeSymbols' = $true}
+
```
Comment (adityapatwardhan) [inline tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1]: Extra newline

**Review by adityapatwardhan — APPROVED** [review]
LGTM with minor comment

**Comment by TravisEz13** [comment]
This is needed to finish high-priority work.  I will merge when tests finish.

**Comment by TravisEz13** [comment]
rebased so I can test with other changes.

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**